diff --git a/meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc b/meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc
index 07e1202a6..7a1044e2f 100644
--- a/meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc
+++ b/meta-sdk/conf/distro/include/fsl-imx-preferred-env.inc
@@ -1,5 +1,5 @@
 PREFERRED_PROVIDER_virtual/bootloader_imx = "u-boot-imx"
-PREFERRED_PROVIDER_virtual/kernel_imx = "linux-imx"
+PREFERRED_PROVIDER_virtual/kernel_imx ?= "linux-imx"
 
 PREFERRED_VERSION_linux-libc-headers_imx ?= "5.10"
 
 