diff --git a/Makefile b/Makefile
index 5e517cd..9f25bb0 100644
--- a/Makefile
+++ b/Makefile
@@ -49,6 +49,7 @@ DOCKER_TARGETS := $(patsubst %,docker-%, $(TARGETS))
 .PHONY: $(TARGETS) $(DOCKER_TARGETS)
 
 GOOS ?= linux
+GOARCH ?= arm64
 
 binaries: cmds
 ifneq ($(PREFIX),)
@@ -56,14 +57,14 @@ cmd-%: COMMAND_BUILD_OPTIONS = -o $(PREFIX)/$(*)
 endif
 cmds: $(CMD_TARGETS)
 $(CMD_TARGETS): cmd-%:
-	GOOS=$(GOOS) go build -ldflags "-s -w" $(COMMAND_BUILD_OPTIONS) $(MODULE)/cmd/$(*)
+	GOOS=$(GOOS) GOARCH=$(GOARCH) go build -ldflags "-s -w" $(COMMAND_BUILD_OPTIONS) $(MODULE)/cmd/$(*)
 
 build:
-	GOOS=$(GOOS) go build ./...
+	GOOS=$(GOOS) GOARCH=$(GOARCH) go build ./...
 
 examples: $(EXAMPLE_TARGETS)
 $(EXAMPLE_TARGETS): example-%:
-	GOOS=$(GOOS) go build ./examples/$(*)
+	GOOS=$(GOOS) GOARCH=$(GOARCH) go build ./examples/$(*)
 
 all: check test build binary
 check: $(CHECK_TARGETS)
